t/ui/13-admin.t

```diff
@@ -347,19 +347,22 @@ subtest 'job property editor' => sub() {
 
     subtest 'edit some properties' => sub() {
         # those keys will be appended
-        $driver->find_element('#editor-name',                           'css')->send_keys(' has been edited!');
-        $driver->find_element('#editor-size-limit',                     'css')->send_keys('0');
-        $driver->find_element('#editor-keep-important-results-in-days', 'css')->send_keys('500');
-        $driver->find_element('#editor-description',                    'css')->send_keys('Test group');
-        $driver->find_element('p.buttons button.btn-primary',           'css')->click();
+        $driver->find_element_by_id('editor-name')->send_keys(' has been edited!');
+        my $ele = $driver->find_element_by_id('editor-size-limit');
+        $ele->click();
+        $ele->send_keys('1000');
+        $ele = $driver->find_element_by_id('editor-keep-important-results-in-days');
+        $ele->click();
+        $ele->send_keys('500');
+        $driver->find_element_by_id('editor-description')->send_keys('Test group');
+        $driver->find_element_by_css('p.buttons button.btn-primary')->click();
         # ensure there is no race condition, even though the page is reloaded
         t::ui::PhantomTest::wait_for_ajax;
 
-        # now reload the page to see if we succeeded
-        $driver->get($driver->get_current_url());
+        $driver->refresh();
```

Comment:
If you are wondering why we used `$driver->get($driver->get_current_url());` before: refresh() just didn't work.
But apparently it works now, so updating Selenium was definitely a good idea.